src/searchReplace.tsx

```diff
@@ -301,6 +303,20 @@ function createTreeView(
         }}
       >
         <span title={file.path}>{file.path}</span>
+        <Button
+          title="replace a particular result" // todo: use an index for unique title per file name
```

Comment:
don't use unique title.
The button is next to a filepath that provides its context.

If you want to use tick, remember those are Javascript string template that need to be evaluated. So you can not set directly string in React. You need to wrap them in curly braces:

```ts
title={`templated ${str}`}
```

Reply:
thanks